exchanges/kraken.py

```diff
@@ -25,15 +30,20 @@ def order_info(self, order_id):
     def trades_history(self, asset_pair, start_time, finish_time):
         pass
 
-    def _public_query(self, query_name, parameters=None):
+    def _public_query(self, query_name, data=None):
         """Queries public data on Kraken"""
 
         # Request public data. This can raise an exception
         # if there is a connection error or if the request times
         # out. In addition, raise an exception if the status is not
         # OK.
         url = '{}public/{}'.format(self.url, query_name)
-        response = requests.get(url, timeout=self.timeout)
+        response = requests.post(url, data=data, timeout=self.timeout)
```

Comment:
Your are right, works with the keyword `params`. Strange that get accepts `data`, but does not seem to use it.